### PR TITLE
Explain how to add InnerSource Stories in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by spier**

### CODE REVIEW

**Comment by spier** [comment]
@yuhattor I noticed that in your case in #683 you did not add the story of nifty to all translated versions of the site.
How did you determine to which translated versions to add the story to?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,3 +11,12 @@ Contributions to this repo are welcome. Please follow the following guidelines:
...
 * For i18n contributors, we recommend reading [the instructions on how it works](i18n.md).
+
+## Adding "InnerSource Stories"
+
+When adding a new organization on the [InnerSource Stories](https://innersourcecommons.org/stories/) page, please follow these steps:
+
+* Provide some evidence of the use of InnerSource practices at the given organization. Optimally an online reference that you can link to.
+* Get confirmation from the organization that they are happy to be listed on the Stories page.
+
+Here an [example PR](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/683) demonstrating this.
```
Comment (jeffabailey) [inline CONTRIBUTING.md]:
```suggestion
Here's an [example PR](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/683) demonstrating this. 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,3 +11,12 @@ Contributions to this repo are welcome. Please follow the following guidelines:
 * Follow the GitHub Guidelines on creating a pull request.
```
Comment (jeffabailey) [inline CONTRIBUTING.md]:
```suggestion
* Follow the [GitHub Pull Request Guidelines](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to create a pull request.
```

**Review by jeffabailey — CHANGES_REQUESTED** [review]

**Comment by spier** [comment]
@jeffabailey thanks for the review. I incorporated your changes. 👍

**Review by rrrutledge — APPROVED** [review]
Looks great ❗  Thank you!

**Comment by spier** [comment]
@yuhattor I already merged this but if you want to add further details related to the translation pages for the InnerSource Stories, feel free to do so.

**Comment by yuhattor** [comment]
I'm late to the party. But it's looking good!!
Thank you!

**Comment by yuhattor** [comment]
For i18n, I think this line is all we need for now👍

> * For i18n contributors, we recommend reading [the instructions on how it works](i18n.md).

**Pull request merged**